feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt

```diff
@@ -0,0 +1,130 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import com.iteneum.dashboard.R
+import com.iteneum.designsystem.components.LPGenericElevatedCard
+import com.iteneum.designsystem.components.LpBadgeButton
+import com.iteneum.designsystem.components.LpGenericCard
+import com.iteneum.designsystem.components.getRandomColor
+import com.iteneum.designsystem.theme.LPTypography
+import com.iteneum.designsystem.theme.LeasePertTheme
+import kotlin.random.Random
+
+
+/**
+ * [DashboardView] Contains a welcome message follow by 2 icons (notifications and profile).
+ * Shows the current balance, the total service and amenities reservations requested.
+ * Also at the view bottom show a event list happening in the same day or events that coming soon.
+ *
+ * @author Jose Rivera
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DashboardView() {
+    val sizes = LeasePertTheme.sizes
+    // TODO mock data, change for real data
+    val username = "Martin"
+    val badgeNumberNotification = "10"
+    val badgeNumberPerson = "20"
+    val showBadgeNotification = true
+    val showBadgePerson = false
+    val currentBalance = "0.00"
+    val currentBalanceCurrency = true
+    val serviceRequest = "1"
+    val amenityReservations = "1"
+    val eventList = listOf("Coffee + Donuts", "Pool party", "Discounts")
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(sizes.regularSize)
+            .verticalScroll(rememberScrollState())
+    ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Text(
+                modifier = Modifier
+                    .align(Alignment.CenterVertically)
+                    .weight(1f),
+                text = stringResource(R.string.welcome_back, username),
+                style = LPTypography.titleMedium,
+            )
+            LpBadgeButton(
+                badgeNumber = badgeNumberNotification,
+                showBadge = showBadgeNotification,
+                imageVector = Icons.Filled.Notifications
+            ) {
+                Log.e("tag", "go to notification")
+            }
+            LpBadgeButton(
+                badgeNumber = badgeNumberPerson,
+                showBadge = showBadgePerson,
+                imageVector = Icons.Filled.Person
+            ) {
+                Log.e("tag", "go to person")
+            }
+        }
+        Spacer(modifier = Modifier.size(sizes.minorRegularSize))
+        LpGenericCard(
+            modifier = Modifier
+                .fillMaxWidth(),
+            title = stringResource(R.string.current_balance),
+            details = stringResource(R.string.go_to_payments),
+            accountNumber = currentBalance,
+            currency = currentBalanceCurrency
+        ) {
+            Log.e("tag", "Go to payments")
```

Comment:
Please add a //TODO comment to tell us which action is going to be executed in this button.